.travis.yml

```diff
@@ -17,7 +17,7 @@ before_script:
   - ./cc-test-reporter before-build
 script:
   - lerna run tslint
-  - lerna run unit-tests --maxWorkers=4
+  - travis_wait lerna run unit-tests --maxWorkers=4
```

Comment:
Let's remove this as it's not related to the PR changes. I have a PR already up too that is configuring travis for publishing and it can go in there.

Reply:
Ok. It did finally allow this test to past.